src/components/SiteHeader/SiteHeader.jsx

```diff
@@ -16,20 +16,47 @@ const HEADER_HIDDEN = {
 /**
  * JSX element for Barnes logos.
  */
-const Logo = ({ size, width, height }) => (
-  <svg className={`a-logo__svg a-logo__svg--${size}`} width={width} height={height} aria-labelledby='logo-title'>
+const Logo = ({ size, width, height, logo, className }) => (
+  <svg className={`a-logo__svg a-logo__svg--${size} ${className ? className : ""}${size === 's' ? "H" : ""}`} width={width} height={height} aria-labelledby='logo-title'>
     <title id='logo-title'>Barnes</title>
-    <use xlinkHref={`#icon--logo-${size}`}></use>
+    <use xlinkHref={`#icon--logo-${size === 's' ? "100H" : logo ? "100" : size}`}></use>
   </svg>
 );
```

Comment:
I think we can do better w/ the logic here, in the logo size arrays, and Logos!
I would refactor this to simplify the render logic:
```
const Logo = ({ size, width, height, logo, className }) => (
  <svg className={`a-logo__svg a-logo__svg--${size} ${className ? className : ""}`} width={width} height={height} aria-labelledby='logo-title'>
    <title id='logo-title'>Barnes</title>
    <use xlinkHref={`#icon--logo-${logo ? logo : size}`}></use>
  </svg>
);
```
And we will move the logic of which logo to use to the logoSizes arrays...